CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Updates default ruby version of GitHub Actions install generator. ([@jamesglover][])
 * Simplify generated GitHub Actions install ([@jamesglover][])
 * Adds boring generator CLI ([@luathn][])
+* Adds Rspec generator. ([@luathn][])
```

Comment:
```suggestion
* Adds RSpec generator. ([@luathn][])
```
As per official guide. Changed the case here to match it. Could you update at other places as well?